src/mgmt/srv_system.c

```diff
@@ -325,21 +298,144 @@ mgmt_svc_step_up_cb(struct ds_rsvc *rsvc)
 static void
 mgmt_svc_step_down_cb(struct ds_rsvc *rsvc)
 {
-	struct mgmt_svc	       *svc = mgmt_svc_obj(rsvc);
-	int			rc;
-
-	svc->ms_step_down = true;
-	ABT_cond_broadcast(svc->ms_distribute_cv);
-	rc = ABT_thread_join(svc->ms_distributord);
-	D_ASSERTF(rc == 0, "%d\n", rc);
-	ABT_thread_free(&svc->ms_distributord);
 }
 
 static void
 mgmt_svc_drain_cb(struct ds_rsvc *rsvc)
 {
 }
 
+struct enum_server_arg {
```

Comment:
Just asking are lines 368-438 here not new - just moving existing code earlier in the file?